File: app/src/main/java/com/bakkenbaeck/sol/ui/BaseActivity.java
package com.bakkenbaeck.sol.ui;


import android.content.Context;
import android.support.v7.app.AppCompatActivity;

import uk.co.chrisjenx.calligraphy.CalligraphyContextWrapper;

public class BaseActivity extends AppCompatActivity {

    @Override
    protected void attachBaseContext(final Context newBase) {
        super.attachBaseContext(CalligraphyContextWrapper.wrap(newBase));
    }
}

File: app/src/main/java/com/bakkenbaeck/sol/view/BaseActivity.kt
package com.bakkenbaeck.sol.view


import android.content.Context
import androidx.appcompat.app.AppCompatActivity

import uk.co.chrisjenx.calligraphy.CalligraphyContextWrapper

open class BaseActivity : AppCompatActivity() {

    override fun attachBaseContext(newBase: Context) {
        super.attachBaseContext(CalligraphyContextWrapper.wrap(newBase))
    }
}
